### PR TITLE
Add ML model and MLOps

### PR DESCRIPTION
**Pull request by WJakubowsk**

### CODE REVIEW

File: src/model/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Use the official Python 3.12 image from Docker Hub
+FROM python:3.12
```
Comment (galkowskim) [inline src/model/Dockerfile:2]: `-slim` version may reduce docker image size

**Review by galkowskim — COMMENTED** [review]
LGTM, only suggestions from `flake8` should be applied

File: src/model/Dockerfile (hunk 1 above)
Comment (WJakubowsk) [reply src/model/Dockerfile]: done.

**Comment by WJakubowsk** [comment]
for some reason, isort in workflow suggests changes while locally it says everything looks good. Suggestion to ignore isort check while merging PR.

**Review by galkowskim — APPROVED** [review]
lgtm

**Pull request merged**